app/models/product.rb

```diff
@@ -1,17 +1,32 @@
 class Product < ApplicationRecord
-  validates :title, :description, :image_url, presence: { message: "can't be blank"}
+
+  validates_with PriceValidator
+  validates :url, presence: true, url: true
```

Comment:
let's have all presence true validations together
let's also use with_options wherever possible

Reply:
why is this this comment not done?